libpod/container_internal.go

```diff
@@ -1166,6 +1167,18 @@ func (c *Container) start() error {
 
 	c.state.State = define.ContainerStateRunning
 
+	if c.config.SdNotifyMode != "ignore" {
+		payload := fmt.Sprintf("MAINPID=%d\n", c.state.ConmonPID)
+		if c.config.SdNotifyMode == "conmon-only" {
```

Comment:
As @rhatdan mentioned in another comment. Using constants would be good. I suggest putting these consts into `libpod/define`, so we can use it in `cmd/podman` without leaking low-level dependencies into the client.